### DISCUSSION TITLE
Why do some sensors show as "unavailable"?

### DISCUSSION THREAD

**Discussion by pdawg17** [moralmunky/Home-Assistant-Mail-And-Packages#544] [Q&A]
FedEx and UPS "delivering" and "delivered" say unavailable. So happy to have found this project btw!

**Comment by firstof9** [comment]
Please review how to [enable debugging](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/wiki/Troubleshooting#enable-debugging) to see what's going on.

**Reply by pdawg17** [reply]
Ok I've been trying to set up debugging but my screen does not look like the example (I'm on latest HA version). After adding the yaml entry, I now see "logger.set_level" as an option but there is no data entry box to add "custom_components.mail_and_packages".

**Reply by firstof9** [reply]
Click the `Go to YAML mode` button.

![image](https://user-images.githubusercontent.com/1105672/138574145-2ad5069a-5988-4885-bc7d-27e21f7faa52.png)
